### PR TITLE
Add recipe for mmh3

### PR DESCRIPTION
**Pull request by johnlees**
This adds a recipe for mmh3, python bindings for murmurhash3

### CODE REVIEW

File: recipes/mmh3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+
+  host:
+    - python
+    - pip
```
Comment (jakirkham) [inline recipes/mmh3/meta.yaml]: Please add the following...

```suggestion
    - pip

  run:
      - python
```

...as Python is a requirement at package install time.

File: recipes/mmh3/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  url: https://github.com/hajimes/{{ name|lower }}/archive/v{{ version }}.tar.gz
+  sha256: 2fe4ee15ca09ea48a7bb6493f2a6d2e7659f33af4d46e6b39ff35d77b549e80f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```
Comment (jakirkham) [inline recipes/mmh3/meta.yaml]: Please also add...

```suggestion
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
```

...as the C compiler is also needed when the C++ compiler is added.

**Comment by jakirkham** [comment]
Thanks @johnlees. This is a nice addition. Made a few comments above, which should hopefully fix the build errors as well.

**Comment by johnlees** [comment]
Thanks for the comments @jakirkham! That seems to have fixed the errors.
Any idea what is happening with the remaining failed test? As far as I can tell it looks like a conda error?

**Pull request merged**